src/test/java/CalendarServletTest.java

```diff
@@ -117,48 +122,40 @@ public void setUp() throws IOException, GeneralSecurityException {
   public void noCalendarEvent() throws IOException, ServletException {
     // Test case where there are no events in the user's calendar
     Mockito.when(calendarClient.getCalendarList()).thenReturn(ONE_CALENDAR);
-    Mockito.when(calendarClient.getCalendarEvents(PRIMARY)).thenReturn(NO_EVENT);
-    List<Event> actual = checkReturn();
-    Assert.assertEquals(NO_EVENT, actual);
+    Mockito.when(calendarClient.getUpcomingEvents(PRIMARY, CURRENT_TIME, END_TIME)).thenReturn(NO_EVENT);
+    Mockito.when(calendarClient.getCurrentTime()).thenReturn(CURRENT_TIME);
+    CalendarClientData actual = getServletResponse();
+    int hour = 60 * 60 * 1000;
```

Comment:
nit: Maybe make this a constant, it is used in multiple tests.

Reply:
Agreed, changed this to a global variable. `Resolved`